vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -170,6 +177,45 @@ export const SelectCheckboxes = {
             },
             deep: true
         }
+    },
+    methods: {
+        onKey(key) {
+            if (!key) return;
+            if (Date.now() - this.keyTimestamp > this.keyTimeout) {
+                this.keyBuffer = "";
+            }
+            this.keyBuffer += key.toUpperCase();
+            this.keyTimestamp = Date.now();
+            this.scrollToBestOption();
+        },
+        scrollToBestOption() {
+            const index = this._items.findIndex(option =>
+                option.label?.toUpperCase().startsWith(this.keyBuffer)
+            );
+
+            if (index === null || index < 0 || index >= this._items.length) return;

```

Comment:
`if (index === -1) return;`

The return value is -1 if not found or >= 0, you shouldn't have all of this conditions, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex#return_value

Reply:
Makes total sense, nice catch!